### PR TITLE
Moves placement of error message in manage person to the left

### PR DESCRIPTION
**Pull request by KaileeHung**
# Ticket Link

[GROUPINGS-1161](https://uhawaii.atlassian.net/browse/GROUPINGS-1161)

# List of squashed commits

- Moves placement of error message in manage person to the left
- Removes flex-break class

# Test Checklist

- [X] Unit Tests Passed:
- [X] Jasmine Tests Passed:
- [X] General Visual Inspection:

### CODE REVIEW

File: src/main/resources/templates/admin.html [hunk 1]
```diff
@@ -203,16 +202,15 @@ <h2 class="card-title mt-md-1 mt-0 mb-1">Manage Person</h2>
...
-
-                            <div ng-show="!errorDismissed2 && !loading && emptySelect"
-                                 class="alert alert-danger form-alert fade show"
-                                 role="alert">
-                                <p th:text="#{screen.message.modal.search.noChecks}"></p>
-                            </div>
-
                         </div>
-
+                        <div class="flex-break"></div>
```
Comment (devgav) [inline src/main/resources/templates/admin.html:206]: This flex-break. Get rid of it here and in our css file.

in bootstrap you can do this: 
<div class="w-100"></div>

**Review by mhodgesatuh — APPROVED** [review]
looks good

**Review by devgav — APPROVED** [review]
Great!. Thanks for updating our UX

**Comment by JorWo** [comment]
I'll make the flex-break to w-100 change in my descriptions PR: https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/pull/628

**Pull request merged**